dts/bindings/auxdisplay/sparkfun,serlcd.yaml

```diff
@@ -32,3 +34,23 @@ properties:
     enum:
       - 2
       - 4
+
+  command-delay:
```

Comment:
use the unit in the property name, `command-delay-ms`, that way one does not have to cross reference the binding documentation to figure it out